trail/query/views.py

```diff
@@ -272,3 +344,56 @@ def get(self, request):
                             serializedMetadata.data,
                             status=status.HTTP_200_OK,
                         )
+
+
+class WcsJoinMetadataQuery(APIView):
+    """
+    Support Query that returns metadata entry that is in the part of sky and matches
+    the metadata params specified by users.
+
+    Attributes
+    ----------
+    metadataDao: MetadataDao
+        Data accessing objects for metadata table
+
+    """
+
+    def __init__(self):
+        self.metadataDao = MetadataDAO()
+
+    def get(self, request):
+        """
+        Returns the Metadata entries that meets both of the sky specified and the params given by the user
+
+        Parameters
+        ----------
+        raLow: float
+            the lower bound for ra
+        raHigh: float
+            the upper bound for ra
+        decLow: float
+            the lower bound for dec
+        decHigh: float
+            the upper bound for dec
+        metadataParams: key-value pairs
+            the key-value pairs in the paramDict that contains the criterias for metadata object
+                Ex.(id = 3, processName=fits)
+
+        Returns
+        -------
+        results: list of metadata objects
+            a list of metadata objects that matches the param specified by users and is in the boundary box.
+
+        Notes
+        -----
+        If users do not specify a dict, the method will return all metadatas.
+        If users do not specify a params, it will return an empty list.
```

Comment:
I'm confused by these notes, I don't immediately see how users specify either thing directly to this method. I guess whatever `request` is should contain a dict as well as whatever params is? This seems like a good test coverage opportunity.